### ISSUE TITLE
Auto bed levelling issue

### ISSUE THREAD

**Issue by meggerja** [NARUTOfzr/ZNP-Robin-Nano-V1.2#12]
Hello, I've got a problem I thing with this auto bed levelling function. I have BLtoutch sensor and no mater what I do the ABL is not probing twice for every point, it does ok two probes for the middle Z offset only. Second fault is maybe not necessary the code but I can't find any other reason: I can level the bed manually and print ok then but if I run the ABL then my printer has false correction on X axis, it is lifting the nozzle at X0-X10 about 0.2 higher than X200-X210. I've got a different sensor offset position (X=0, Y=-25, Z=-3.7) witch I set in the config file but I don't think it should be a problem.
[elegoo.txt](https://github.com/NARUTOfzr/ZNP-Robin-Nano-V1.2-V1.3/files/11232758/elegoo.txt)

**Comment by NARUTOfzr** [comment]
Don't know where you installed bltouch? Can you take a picture to see?




---Original---
From: ***@***.***&gt;
Date: Fri, Apr 14, 2023 20:27 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [NARUTOfzr/ZNP-Robin-Nano-V1.2-V1.3] Auto bed levelling issue (Issue#12)




 
Hello, I've got a problem I thing with this auto bed levelling function. I have BLtoutch sensor and no mater what I do the ABL is not probing twice for every point, it does ok two probes for the middle Z offset only. Second fault is maybe not necessary the code but I can't find any other reason: I can level the bed manually and print ok then but if I run the ABL then my printer has false correction on X axis, it is lifting the nozzle at X0-X10 about 0.2 higher than X200-X210. I've got a different sensor offset position (Y=0, X=-25, Z=-3.7) witch I set in the config file but I don't think it should be a problem.
 elegoo.txt

**Comment by meggerja** [comment]
It is from this project:
https://www.printables.com/model/284757-elegoo-neptune-2-2s-2d-fan-upgrade-w-bl-touch
![image](https://user-images.githubusercontent.com/19674998/232124098-b20e0582-a3f5-40df-ac5c-f512ee9083c3.png)

**Comment by meggerja** [comment]
Just watched another video on YT and spotted the probing is only once per spot, so this is actually not a fault. My mistake

**Comment by NARUTOfzr** [comment]
Yes, each point is only probed once. I made some changes. This should improve.
![111-2023-04-15 095844](https://user-images.githubusercontent.com/64059773/232177733-edb05d6f-7537-463a-9812-9445b18cdb16.jpg)
[1-elegoo.txt](https://github.com/NARUTOfzr/ZNP-Robin-Nano-V1.2-V1.3/files/11238054/1-elegoo.txt)
